### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by aprilyang**

### CODE REVIEW

File: docusaurus.footer.config.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                        </div>
+                    `,
+                },
+            ],
+        },
+        {
+            title: 'Projects',
+            items: [
+                {label: 'go-storage', href: 'https://github.com/beyondstorage/go-storage'},
+                {label: 'go-ftp-server', href: 'https://github.com/beyondstorage/go-ftp-server'},
```
Comment (Xuanwo) [inline docusaurus.footer.config.js:33]: Let's remove it from our projects for now.

File: docusaurus.navbar.config.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        alt: 'My Site Logo',
+        src: 'img/logo.svg',
+    },
+    items: [
+        {
+            label: 'Projects',
+            position: 'left',
+            items: [
+                {label: 'go-storage', href: 'https://github.com/beyondstorage/go-storage'},
+                {label: 'go-ftp-server', href: 'https://github.com/beyondstorage/go-ftp-server'},
```
Comment (Xuanwo) [inline docusaurus.navbar.config.js:13]: ditto.

File: i18n/zh-CN/code.json [hunk 3]
```diff
@@ -190,5 +190,14 @@
   "theme.SearchBar.label": {
     "message": "Search",
     "description": "The ARIA label and placeholder for search button"
+  },
+  "Open": {
```
Comment (Xuanwo) [inline i18n/zh-CN/code.json:194]: Don't add translations by hand. We use `crowdin` to maintain them.

File: i18n/zh-CN/docusaurus-theme-classic/footer.json [hunk 4]
```diff
@@ -1,6 +1,46 @@
...
+  "link.title.Projects": {
+    "message": "项目",
+    "description": "The title of the footer links column with title=Projects in the footer"
+  },
+  "link.title.Resources": {
+    "message": "资源",
+    "description": "The title of the footer links column with title=Resources in the footer"
+  },
+  "link.title.Community": {
+    "message": "社区",
```
Comment (Xuanwo) [inline i18n/zh-CN/docusaurus-theme-classic/footer.json:15]: ditto.

File: i18n/zh-CN/docusaurus-theme-classic/navbar.json [hunk 5]
```diff
@@ -1,18 +1,34 @@
...
   "item.label.Community": {
     "message": "社区",
     "description": "Navbar item with label Community"
   },
   "item.label.Blog": {
     "message": "博客",
     "description": "Navbar item with label Blog"
+  },
+  "item.label.Projects": {
+    "message": "项目",
```
Comment (Xuanwo) [inline i18n/zh-CN/docusaurus-theme-classic/navbar.json:15]: ditto.

File: docusaurus.footer.config.js (hunk 1 above)
Comment (aprilyang) [reply docusaurus.footer.config.js:33]: remove go-storage and go-ftp-server ?

Comment (Xuanwo) [reply docusaurus.footer.config.js:33]: remove `go-ftp-server` and keep `go-storage`

**Comment by aprilyang** [comment]
@Xuanwo Fixed.

**Pull request merged**